packages/memory-storage/src/fs/key-value-store/fs.ts

```diff
@@ -2,6 +2,7 @@ import { ensureDir } from 'fs-extra';
 import { readFile, rm } from 'node:fs/promises';
 import { dirname, resolve } from 'node:path';
 import { basename } from 'node:path/win32';
+import { AsyncQueue } from '@sapphire/async-queue';
```

Comment:
undeclared dependency, its not enough to have this in core if you want to import it here

Reply:
Already declared :D
![Code - 2023-04-07 at 12 33 21@2x](https://user-images.githubusercontent.com/17960496/230585001-de80e9b5-52e6-4dc1-8bbc-c7bd0dd95fd5.png)